content/influxdb/cloud-dedicated/admin/databases/update.md

```diff
@@ -0,0 +1,138 @@
+2.  Run the `influxctl database update` command and provide the following:
+
+    - Database name
+    - _Optional_: Database [retention period](/influxdb/cloud-dedicated/admin/databases/#retention-periods)
+      _(default is infinite)_
+    - _Optional_: Database table (measurement) limit _(default is 500)_
+    - _Optional_: Database column limit _(default is 250)_
+
+{{% code-placeholders "DATABASE_NAME|30d|500|200" %}}
+```sh
+influxctl database update DATABASE_NAME \
+  --retention-period 30d \
+  --max-tables 500 \
+  --max-tables 250
+```
+{{% /code-placeholders %}}
+
+{{% warn %}}
+#### Database names can't be updated
+
+The `influxctl database update` command uses the database name to identify which
+database to apply updates to. The database name itself can't be updated.
+{{% /warn %}}
+
+- [Retention period syntax](#retention-period-syntax)
+- [Database naming restrictions](#database-naming-restrictions)
+- [InfluxQL DBRP naming convention](#influxql-dbrp-naming-convention)
+- [Table and column limits](#table-and-column-limits)
+
+## Retention period syntax
+
+Use the `--retention-period` flag to define a specific
+[retention period](/influxdb/cloud-dedicated/admin/databases/#retention-periods)
+for the database.
+The retention period value is a time duration value made up of a numeric value
+plus a duration unit. For example, `30d` means 30 days.
+A zero duration retention period is infinite and data will not expire.
+The retention period value cannot be negative or contain whitespace.
+
+{{< flex >}}
+{{% flex-content %}}
+
+##### Valid durations units include
+
+- **m**: minute
+- **h**: hour
+- **d**: day
+- **w**: week
+- **mo**: month
+- **y**: year
+
+{{% /flex-content %}}
+{{% flex-content %}}
+
+##### Example retention period values
+
+- `0d`: infinite/none
+- `3d`: 3 days
+- `6w`: 6 weeks
+- `1mo`: 1 month (30 days)
+- `1y`: 1 year
+- `30d30d`: 60 days
+- `2.5d`: 60 hours
+
+{{% /flex-content %}}
+{{< /flex >}}
+
+## Database naming restrictions
+
+Database names must adhere to the following naming restrictions:
+
+- Must contain two or more characters
+- Cannot start with an underscore (`_`)
+- Cannot contain whitespace characters, double quotes (`"`), or percent signs (`%`)
+
+## InfluxQL DBRP naming convention
+
+In InfluxDB 1.x, data is stored in [databases](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#database)
+and [retention policies](/{{< latest "influxdb" "v1" >}}/concepts/glossary/#retention-policy-rp).
+In InfluxDB Cloud Dedicated, databases and retention policies have been merged into
+_databases_, where databases have a retention period, but retention policies
+are no longer part of the data model.
+Because InfluxQL uses the 1.x data model, a database must be mapped to a v1
+database and retention policy (DBRP) to be queryable with InfluxQL.
+
+**When naming a database that you want to query with InfluxQL**, use the following
+naming convention to automatically map v1 DBRP combinations to a database:
```

Comment:
I realize this section is already under create, but I have questions:

* This reads like I have to have a `/retention-policy` suffix on the database name to use influxql. Is that right?
* How does someone know what retention policy name to use? Are these figured out automatically somehow?